geomesa-accumulo/geomesa-accumulo-jobs/pom.xml

```diff
@@ -61,77 +70,9 @@
             <artifactId>slf4j-log4j12</artifactId>
         </dependency>
         <dependency>
-            <groupId>org.specs2</groupId>
-            <artifactId>specs2-core_${scala.binary.version}</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>org.specs2</groupId>
-            <artifactId>specs2-junit_${scala.binary.version}</artifactId>
+            <groupId>log4j</groupId>
+            <artifactId>log4j</artifactId>
         </dependency>
     </dependencies>
 
-    <profiles>
-        <profile>
-            <id>assemble</id>
```

Comment:
*blinks*  Was this being used?!  Good eye to remove it!

Reply:
eh, we had docs on it... but it's one of those things that we never test, so it likely doesn't even work anymore.

Reply:
Are the docs gone?

Reply:
yeah i deleted them